applications/tari_validator_node_cli/src/command/transaction.rs

```diff
@@ -289,7 +289,7 @@ pub async fn submit_transaction(
         .sign(&key.secret_key)
         .build();
 
-    if transaction.meta().involved_shards().is_empty() {
+    if transaction.meta().involved_shards().is_empty() && transaction.meta().max_outputs() == 0 {
```

Comment:
Since the VNs attempt to determine outputs, we should still be able to submit a transaction with 0 outputs (max_outputs should be named num_new_outputs)